src/components/v5/common/ActionSidebar/partials/hooks.ts

```diff
@@ -57,5 +61,9 @@ export const useSubmitButtonDisabled = () => {
     return true;
   }
 
+  if (selectedAction === ACTION.UPGRADE_COLONY_VERSION && !canUpgrade) {
+    return true;
+  }
+
```

Comment:
This is starting to drag with all the if cases (2 now, more on the way most likely), so please refactor this as a `switch` statement

Reply:
Refactored this as a switch statement.